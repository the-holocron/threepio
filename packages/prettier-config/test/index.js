/* eslint-env jest */
const config = require('../prettier.config.json');

describe('Preset', () => {
	describe('Prettier', () => {
		it('should render the configuration', () => expect(config).toMatchSnapshot());
	});
});
