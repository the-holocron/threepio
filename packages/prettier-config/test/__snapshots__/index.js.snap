// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preset Prettier should render the configuration 1`] = `
Object {
  "arrowParens": "always",
  "bracketSpacing": true,
  "jsxBracketSameLine": false,
  "printWidth": 120,
  "quoteProps": "as-needed",
  "semi": true,
  "tabWidth": 4,
  "trailingComma": "es5",
  "useTabs": true,
}
`;
